lib/Hakyll/Web/Feed.hs

```diff
@@ -176,8 +220,32 @@ renderAtom :: FeedConfiguration       -- ^ Feed configuration
 renderAtom = renderAtomWithTemplates atomTemplate atomItemTemplate
 
 
+--------------------------------------------------------------------------------
+-- | Render a JSON feed with a number of items.
+renderJsonFeed :: FeedConfiguration       -- ^ Feed configuration
+           -> Context String          -- ^ Item context
+           -> [Item String]           -- ^ Feed items
+           -> Compiler (Item String)  -- ^ Resulting feed
+renderJsonFeed = renderJsonFeedWithTemplates jsonFeedTemplate jsonFeedItemTemplate
+
+
 --------------------------------------------------------------------------------
 -- | Copies @$updated$@ from @$published$@ if it is not already set.
 makeItemContext :: String -> Context a -> Context a
 makeItemContext fmt context = mconcat
     [context, dateField "published" fmt, dateField "updated" fmt]
+
+
+--------------------------------------------------------------------------------
+-- | Escape the string with the usual Haskell escape conventions while leaving
+-- the non-ASCII characters intact.
+escapeString :: String -> String
```

Comment:
This function renders a "delete" character as `\DEL`, which is not valid JSON. I think it'd be best to model this after [RFC 8259 §7](https://www.rfc-editor.org/rfc/rfc8259#section-7). Can you please do that?

Reply:
I modified the escaping logic to match the RFC, can you please take another look?